async_task.js

```diff
@@ -0,0 +1,88 @@
+/* 1. Створити модуль api_helper.js, в якому будуть реалізовані функції посилання get та post запитів
+та десеріалізація респонсу у json об'єкт, який вони будуть повертати (при реалізації поста зверніть увагу на хедери, 
+які вказані у інструкції під блоком Post with JSON). Для створення запитів використати пакет
+node-fetch, інструкця по використанню тут https://www.npmjs.com/package/node-fetch?activeTab=readme, для 
+реалізації асинхронності використати async/await */
+
+import * as request from "./api_helper/api_helper.js";
+
+const baseUrl = "https://jsonplaceholder.typicode.com/posts/";
+
+
+// /* 2. Реалізувати функцію getPostsByUsedId, яка буде знаходити пости з ресурсу https://jsonplaceholder.typicode.com/posts та робити 
+// вибірку постів за id користувача. Також у всіх відфільтрованих постів повинна бути відсутня властивість title */
+
+// async function getPostsByUsedId(url, userId) {
+// // Ваш код
+
+// try {
+//     const getResponse = await fetch(url);
+//     const result = await getResponse.json();
+//     const filteredPosts = result
+//     .filter(post => post.userId === userId)
+//     .map(post => {
+//         const { title, ...rest } = post;
+//         delete rest.title; // Видалення властивості title
+//         return rest;
+//     });
+//     return filteredPosts;
+
+// } catch (error) {
+//     console.error('Error during GET request:', error);
+//     throw error;
+// }
+
+// }
+
+// const posts = await getPostsByUsedId(baseUrl, 5);
+// console.log(posts); // маємо тільки пости юзера з id = 5, у яких нема title
+
+// /* 3. Реалізувати функцію createNewPost, яка буде створювати новий пост на ресурсі https://jsonplaceholder.typicode.com/posts */
+// const body = {
+//   userId: 155,
+//   title: "New title",
+//   body: "New body",
+// };
+// async function createNewPost(url, body) {
+//     // Ваш код
+//     try {
+//             const makePostRequest = await fetch(url,{
```

Comment:
```suggestion
//             const makePostRequest = await request.postRequest(url,{
```
тут також варто було використати готову функцію

Reply:
@Art-auto 
тут в мене після заміни 49 строки в консолі та сама помилка що і з get запитом
Error during POST request: TypeError: makePostRequest.json is not a function

![image](https://github.com/qarber/hillel_qa_automation/assets/147742705/8d69da7f-4ba3-4c11-b67d-362c21e9609b)